README.md

```diff
@@ -4,8 +4,46 @@
 
 > Feel free to explore my work
 
+![screenshot](https://drive.google.com/file/d/1Biwj7V5NOQTpdYbSaQnY2ze8rRn8ODrP/view?usp=sharing)
+
 ## Built With
 
 - HTML
 - CSS
-- JS
\ No newline at end of file
+- JS
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+
+N/A
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@Lino09](https://github.com/Lino09)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] It seems that the path to your MIT.md is broken, since I am getting a 404(Page not found) error when I click on it, kindly let's fix this link 👍